doc/parsing.rst

```diff
@@ -247,6 +203,48 @@ longer ones as "null" alignments. The maximal size of ignored *gaps* is set by
 the ``--max-inter-align-gap`` flag (by default, 20bp).
 
 
+Parse2
+-------------------------
+
+If your Hi-C has long reads, you may want to report all the alignments in the reads with ``pairtools parse2``.
```

Comment:
maybe specify read length where it would benefit? e.g.
If your Hi-C has long reads (>50bp). 

Would this work on nanopore reads or not? Maybe good to mention somewhere

Reply:
150 bp allows you to save 10% of the simple Hi-C library on DpnII. On Nanopore this won't work for now. Good points, will mention!

Reply:
Resolved by https://github.com/open2c/pairtools/pull/99